templates/source/sourceTemplate.md

```diff
@@ -0,0 +1,35 @@
+# Event Source for **\_\_\_\_**
+
+This event target receives .......
+
+## Prerequisites
+
+- unordered list out prerequisites
+- unordered list out prerequisites
+- unordered list out prerequisites
+
+##### This should be about a prerequisites
+
+1. number list of each step for the prerequisite
+2. number list of each step for the prerequisite
+3. number list of each step for the prerequisite
+
+Or if there are no steps, you can add details about each one.
+
+##### This should be about a prerequisites
+
+1. number list of each step for the prerequisite
+2. number list of each step for the prerequisite
+3. number list of each step for the prerequisite
+
+Or if there are no steps, you can add details about each one.
+
+## Deploying an Instance of the Source
+
+Open the Bridge creation screen and add a source of type **\_\_**
+
+Continue adding descriptions and steps needed.
+
+## Event Types
+
+Add the event type info here.
```

Comment:
I think we need to templatize that section too.

I'd be in favor of something short and concise, like the event type emphasized (title and `backticks` maybe?) followed by a single sentence describing the event. Link to a schema whenever possible like Pablo said, this is critical IMO.

Reply:
Okay I will write something up. Thanks for the help. That is why I asked for help creating these. I have no idea what info needs to be here and what does not  :)

Reply:
Okay I will write something up. Thanks for the help. That is why I asked for help creating these. I have no idea what info needs to be here and what does not :)

Reply:
No need for a justification or apology 👍 Code reviews are for discussing changes, which is what we are doing here.